.github/workflows/lint-test.yml

```
@@ -100,31 +131,31 @@ jobs:
           python-version: ${{ matrix.python-version }}
 
       - name: Install dependencies
-        run: pdm install -d -Gspeed -Gdocs -Gvoice
+        run: nox -s pyright --no-venv --install-only --force-python ${{ steps.setup-env.outputs.python-version }}
```

```suggestion
        # `--no-venv` to install in the main pdm venv instead of nox's pyright-specific one
        run: nox -s pyright --no-venv --install-only --force-python ${{ steps.setup-env.outputs.python-version }}
```
(assuming this is correct?)